liqd_product/assets/scss/components/_pagination.scss

```diff
@@ -1,33 +1,32 @@
 .pagination {
-    @include clearfix;
-    display: inline-block;
-    padding: 0;
-    padding-right: 1px;
-    background-color: $body-bg;
+    margin-top: 1rem;
 }
 
 .pagination-item {
-    float: left;
-    display: inline-block;
+    min-width: 40px;
+
+    @media screen and (min-width: $breakpoint-xs) {
+        min-width: 100px;
+    }
 
     a {
-        display: inline-block;
-        min-width: 1.4em + 2 * 0.5em;
-        margin-right: -1px;
+        background-color: $body-bg;
```

Comment:
I would set the background to $bg-light, I think the body-bg is the old background colour? and larger min width on the large screen buttons will make them look more even (i could have remembered that yesterday but brain was dead)